src/components/DMAComponent.js

```diff
@@ -42,17 +43,22 @@ function DMAComponent({ device }) {
     'Channel 1', 'Channel 2', 'Channel 3', 'Channel 4',
   ];
 
+  const warn = 0.001; // TBD
```

Comment:
What are these tbd ?

Reply:
To be defined... As for now I don't know which values I can use for yellow and red colors